app/forms/candidate_interface/course_choices/course_selection_wizard.rb

```diff
@@ -0,0 +1,35 @@
+module CandidateInterface
+  module CourseChoices
+    class CourseSelectionWizard < DfE::Wizard::Base
+      # application_choice is only used in edit and update
+      attr_accessor :current_application, :application_choice
+
+      steps do
+        [
+          { do_you_know_the_course: CourseChoices::DoYouKnowTheCourseStep },
+          { go_to_find_explanation: CourseChoices::GoToFindExplanationStep },
+          { provider_selection: CourseChoices::ProviderSelectionStep },
+          { which_course_are_you_applying_to: CourseChoices::WhichCourseAreYouApplyingToStep },
+          { duplicate_course_selection: CourseChoices::DuplicateCourseSelectionStep },
+          { reached_reapplication_limit: CourseChoices::ReachedReapplicationLimitStep },
+          { full_course_selection: CourseChoices::FullCourseSelectionStep },
+          { closed_course_selection: CourseChoices::ClosedCourseSelectionStep },
+          { course_study_mode: CourseChoices::CourseStudyModeStep },
+          { course_site: CourseChoices::CourseSiteStep },
+          { find_course_selection: CourseChoices::FindCourseSelectionStep },
+          { course_review: CourseChoices::CourseReviewStep },
+        ]
+      end
+
+      store CandidateInterface::CourseSelectionStore
```

Comment:
Not a blocker, but to keep the name spacing consistent, should this be `CandidateInterface::CourseChoices::CourseSelectionStore`?

Reply:
This was a worthy change 👌